src/sqlfluff/core/rules/std/L031.py

```diff
@@ -59,6 +59,7 @@ def _eval(self, segment, **kwargs):
                 return None
 
             from_expression = from_clause_segment.get_child("from_expression")
+            from_expression_element = None
```

Comment:
Fixing an undefined variable (unrelated to #778, but discovered while working on it)